src/schema/objects/metadata.yaml

```diff
@@ -3392,6 +3392,24 @@ SubjectArtefactDescription:
     If this field is set to `"n/a"`, it will be interpreted as absence of major
     source of artifacts except cardiac and blinks.
   type: string
+TablePosition:
+  name: TablePosition
+  display_name: Table Position
+  description: |
+    The table position, relative to an implementation-specific reference point,
+    often the isocenter. Values must be an array (1x3) of three distances in
+    millimeters in absolute coordinates (world coordinates). The
+    absolute coordinates are respectively right, anterior and superior (RAS+)
+    as if it was acquired in head first supine (HFS) orientation. In
+    other words, moving the table into the scanner will always increase the 3rd
+    value in the array, regardless of the patient positioning (feet
+    first for example). The origin is defined by the image affine.
```

Comment:
Really? I thought that the image affine origin was in reference to the scanner isocenter (https://nipy.org/nibabel/coordinate_systems.html). Let's imagine two scenarios: 
1. Patient is installed, isocenter is set around the eyes, table goes in, image is acquired with FOV centered around the eyes, and, let's say, `TablePosition(3)=x`. 
2. Patient is installed, isocenter is set around the chin (which is, let's say -100mm from the eyes), image is acquired with FOV centered around the chin, I would expect `TablePosition(3)=x-100`, so not the same as in the previous scenario, even though both images are acquired with the FOV at isocenter (ie: `affine(3,4)=0`). 

I have to admit it is easy to get confused with this, so it is possible that my demonstration has a glitch somewhere... but in any case, I would find it useful to give similar example scenario to explain how TablePosition is calculated in the BIDS definition.

Reply:
Great scenario. I took a look at previous acquisitions (brain and spinal cord scans), I almost always see the `TablePosition` DICOM tag (at least on Siemens and Philips) at <|1| on the first scan (localizer). This suggests to me that the tag is an offset once it has been set to isocenter and does not depend on where it was set on the patient.

> Really? I thought that the image affine origin was in reference to the scanner isocenter (https://nipy.org/nibabel/coordinate_systems.html). Let's imagine two scenarios:

I believe the affine origin is isocenter if the TablePosition = [0,0,0], otherwise it is not. Maybe `IsocenterOffset`, `TablePositionOffset` is a better name tag?